python/Cargo.toml

```
@@ -27,3 +27,6 @@ uuid = "1.3.0"
 
 [build-dependencies]
 prost-build = "0.11"
+
+[features]
+opq = ["lance/opq"]
```

Let's keep this on by default.

```suggestion
[features]
default = ["opq"]
opq = ["lance/opq"]
```